CMakeLists.txt

```diff
@@ -135,7 +135,9 @@ if(NOT CMAKE_CROSSCOMPILING)
         if(ARCH STREQUAL "i386")
             add_definitions(/D_X86_ /D__i386__ /DWIN32 /D_WINDOWS)
         elseif(ARCH STREQUAL "amd64")
-            add_definitions(-D_AMD64_ -D__x86_64__ /DWIN32 -D_WINDOWS)
+            # '__x86_64': This is NOT a typo.
+            # See https://software.intel.com/en-us/forums/topic/404643
```

Comment:
Dead link

Reply:
It seems it was just moved from [here](https://github.com/reactos/reactos/pull/4139/files#diff-4a0d335f415e68b8558820e21f06204c347f48677fda1b26b890f12dd5fed4e8L362). But yes, can confirm, I can't retrieve it, even not archived in Web Archive.

Reply:
https://community.intel.com/t5/Software-Archive/Origins-of-X86-64-macro-used-in-header-files-of-GCC-and-Intel/td-p/967540?profile.language=ko Maybe some info here?

Reply:
Found: https://community.intel.com/t5/Intel-C-Compiler/32-bit-intel/td-p/964398 (the link posted above has the exact thread name mentioned, that's how I found it)